package.json

```diff
@@ -477,6 +477,35 @@
                 "url": "http://json.schemastore.org/proxies"
             }
         ],
+        "problemPatterns": [
+            {
+                "name": "func",
+                "kind": "file",
+                "regexp": "^.*Missing.*AzureWebJobsStorage.*(local.settings.json).*$",
```

Comment:
Consider more general, e.g.:

Missing value for \w+ in ([\w\.]+\w)

Reply:
I don't think that regexp applies though. Like I mentioned, the func cli has REALLY inconsistent formatting for errors. I don't want to accidentally pick up things that aren't errors (or that are warnings instead of errors)

I picked one of the most common errors and made it work for that, but we'll have to file an issue on the func cli if we want to actually have a generic regexp